frontend/src/modules/displays/MultiStepWizard.tsx

```diff
@@ -25,6 +30,26 @@ import { commitRequest, commitDraftRequest } from '../../store/controller/action
 import store from '../../store';
 import { DASHBOARD_ROUTE } from '../navigation/constants';
 
+/**
+ * @typedef Data
+ *
+ * Interface for Data
+ *
+ * @prop {string} stateKey Unique identifier for the value
+ * @prop {string} label Label for the value
+ * @prop {string} units Unit measurement of the value
+ * @prop {boolean} isAlarmEnabled Config to set if value is Alarm type
+ * @prop {boolean} isSetvalEnabled Config to set if value is Set type
+ * @prop {number | null} committedSetting Current value
+ * @prop {number[]} alarmValues Alarm range values
```

Comment:
What is the relationship between `alarmValues` and `alarmLimitMin` and `alarmLimitMax` and `alarmValuesActual`?

Reply:
`alarmValues` - AlarmValues that user modifies in the multistep popup
`alarmLimitMin` - Allowed minimum range of alarm (0 by default)
`alarmLimitMax` - Allowed max range of alarm (100 by default)
`alarmValuesActual` - Current Alarm values in redux store for AlarmRange